packages/lesswrong/server/scripts/makeMigrations.ts

```diff
@@ -161,28 +140,80 @@ export const makeMigrations = async ({
 
   for (const collectionName of collectionNames) {
     try {
-      const sql = getCreateTableQueryForCollection(collectionName, forumType);
-
+      const collection = getCollection(collectionName as any);
```

Comment:
Factored out of the `getCreateTableQueryForCollection` function above to give us access to the `Table` object